packages/blocks/src/api/serializer.js

```diff
@@ -255,17 +255,18 @@ export function getCommentDelimitedContent( rawBlockName, attributes, content )
  * Returns the content of a block, including comment delimiters, determining
  * serialized attributes and content form from the current state of the block.
  *
- * @param {Object} block Block instance.
+ * @param {Object} block   Block instance.
+ * @param {Object} options Serialization options.
  *
  * @return {string} Serialized block.
  */
-export function serializeBlock( block ) {
+export function serializeBlock( block, { isNested = false } = {} ) {
```

Comment:
Nit: I think we should try to consolidate our vocabulary around "inner blocks". Do we have many other references of "nested" in code? I know there's a tendency to refer to them this way in casual conversation. Is `isInnerBlock` reasonable as an alternative here?